config/ground-truth-yaml-files/all_features_version_10_no_visit_variant_pcd.yml

```diff
@@ -0,0 +1,2655 @@
+patient:
+  Fever:
+    categories:
+    - biolink:Disease
+    - biolink:DiseaseOrPhenotypicFeature
+    enum: &id133

```

Comment:
Not sure whether we need ```&id133``` here. What is the purpose of  ```&id133``` here?

Reply:
I'm not sure what `&id133` is for, but I suspect they were added as part of the YAML -> Dhall conversion. They do not appear to be causing any trouble, so I left them as is.  If you'd prefer, we can remove them from all of the all_features YAML files.

Reply:
OK. If they don't cause issues, I am fine with keeping them. However, I did notice some enums have the ids similar to what is defined here while some other enums don't. The swagger yaml specification as shown here: https://swagger.io/docs/specification/data-models/enums/ does not appear to have ids as shown here.

Reply:
Actually, I think I remember what the `&` symbol was used for. I believe Hao was using this to reuse elements from the patient files for the visit files and/or to prevent identical patient and visit variables from being tagged as duplicate. I just took a quick look at the version_09, asthma, dili, and pcd all_features YAML files, and they each contain a subset of variables marked `&`, but the id's appear to be different, so I think we can leave things as is. 

Then again, if we want to clean up the files and adhere to best practices, then we probably should delete all of those tags. I'll leave that decision up to you.

Reply:
OK. Good to know. Let's leave it as is then and can deal with any potential issues if they arise down the road.